ff/serving/python_client/sdk/providers.py

```diff
@@ -0,0 +1,58 @@
+providers_to_register = []
+
+class Provider:
+    def __init__(self, name = str, provider_type = str, host = str, db = int, ssl = bool, client_certificate= str, bucket = str):
+        self.name = name
+        self.provider_type = provider_type
+        self.host = host
+        self.db = db
+        self.ssl = ssl
+        self.client_certificate = client_certificate
+        self.bucket = bucket
+      
+
+def register_provider(name, provider_type, host, db, ssl, client_certificate, bucket):
+    provider = Provider(name, provider_type, host, db, ssl, client_certificate, bucket)
+    providers_to_register.append(provider)
+
+def register_redis(*args, **kwargs):
+    register_provider(kwargs['name'], 'Redis', kwargs['host'], kwargs['db'], kwargs['ssl'], kwargs['client_certificate'], kwargs['bucket'])
```

Comment:
We have the redis config on the backend, @Sami1309 wrote it.

You should match these parameters with those.